src/containers/explore/EntityNeighborsContainer.js

```diff
@@ -0,0 +1,171 @@
+/*
+ * @flow
+ */
+
+import React, { useMemo, useState } from 'react';
+
+import styled from 'styled-components';
+import { Map, Set } from 'immutable';
+import { Checkbox, Spinner, Typography } from 'lattice-ui-kit';
+import { useRequestState } from 'lattice-utils';
+import { useSelector } from 'react-redux';
+import { RequestStates } from 'redux-reqseq';
+import type { UUID } from 'lattice';
+import type { RequestState } from 'redux-reqseq';
+
+import EntityNeighborsTable from './EntityNeighborsTable';
+import { EXPLORE_ENTITY_NEIGHBORS } from './actions';
+
+import { DataSetTitle } from '../../components';
+import { EXPLORE } from '../../core/redux/constants';
+import { selectOrgDataSets } from '../../core/redux/selectors';
+
+const ContainerWrapper = styled.div`
+  min-height: 500px;
+`;
+
+const SectionWrapper = styled.section`
+  margin: 15px 0;
+`;
+
+type Props = {
+  isModal :boolean;
+  neighbors :Map;
+  organizationId :UUID;
+};
+
+const EntityNeighborsContainer = ({ isModal, neighbors, organizationId } :Props) => {
```

Comment:
Not sure exactly where to add this comment. I think I found an edge case. When `/neighbors/advanced/ids` returns nothing, then `EntityNeighborsContainer` renders nothing. It should instead show some kind of message to indicate the lack of neighbors.

<img width="1442" alt="Screen Shot 2021-06-07 at 7 49 53 PM" src="https://user-images.githubusercontent.com/440299/121115233-ba20d580-c7c9-11eb-8b1b-4466796ccc42.png">